spec/requests/api/v1/users/update_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe 'PUT api/v1/user', type: :request do
```

Comment:
Since you will have to change your routes to be REST, remember to update this test too.